taca/illumina/HiSeqX_Runs.py

```diff
@@ -62,25 +56,34 @@ def demultiplex_run(self):
 
         ##when demultiplexing SampleSheet.csv is the one I need to use
         ## Need to rewrite so that SampleSheet_0.csv is always used.
-        self.runParserObj.samplesheet  = SampleSheetParser(os.path.join(self.run_dir, "SampleSheet.csv"))
+        self.runParserObj = RunParser(self.run_dir)
```

Comment:
Is there a reason why this is run again instead of `SampleSheetParser` ?

Cause `self.runParserObj = RunParser(self.run_dir)` is already run in [Runs](https://github.com/SciLifeLab/TACA/blob/master/taca/illumina/Runs.py#L119) (?)

Reply:
Hi, this is because if a samplesheet does not exist, `samplesheet` will not be created by running `SampleSheetParser` only.